rflx/model/message.py

```diff
@@ -52,7 +52,7 @@ class Link(Base):
     condition: expr.Expr = expr.TRUE
     length: expr.Expr = expr.UNDEFINED
     first: expr.Expr = expr.UNDEFINED
-    location: Optional[Location] = None
+    location: Optional[Location] = dataclass_field(default=None, repr=False)
```

Comment:
What is the reason for this change?

Reply:
That makes failing tests more readable, as `location` will not be part of the object representation. We do the same in `common.Base`, but `dataclass` overwrites `__repr__`.